documentation/templates/documentation/partials/block.html

```diff
@@ -166,7 +166,12 @@ <h2>Tips</h2>
 {% if code_block.ext_doc %}
     <div id="external_doc">
         <h2>Additional Information</h2>
-        <p>From <a href="{{ code_block.ext_doc }}" taraget="_blank">{{ code_block.ext_doc }}</a></p>
-        <iframe src="{{ code_block.ext_doc }}" width="100%" height="1000px" frameborder="none"></iframe>
+        <!-- dont render iframe for w3schools content, see https://codedotorg.atlassian.net/browse/LP-1424 -->
```

Comment:
I would either explain the rationale here, or link to the PR and put the info in the PR description, since our JIRA tickets aren't visible to those outside the org and we might change tools in the future.